packages/styles/components/_c.select.scss

```diff
@@ -0,0 +1,96 @@
+$select-border-radius : get-border-radius('m');
+$select-border-width : get-border('s');
+
+.mc-select {
+  border: $select-border-width solid $color-select-border;
+  border-radius: $select-border-radius;
+  display: block;
+
+  @include set-font-scale('05');
+
+  background: $color-select-background;
+  color: $color-select-text;
+  position: relative;
+  width: 100%;
+
+  /* autoprefixer: ignore next */
+  -webkit-transform: translateZ(0);
+
+  &:hover,
+  &.is-hover {
+    border: $select-border-width solid $color-select-hover-border;
+  }
+
+  &:focus,
```

Comment:
not sure the div understand focus

Reply:
Instead you can use `:focus-within` which will catch a focus event on a element inner the div wrapper. I supposed that what you want.

Reply:
`:focus-within` is not supported by ie and edge.